ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/ldesfragment/valueobjects/LdesFragmentIdentifier.java

```diff
@@ -31,7 +31,8 @@ public List<FragmentPair> getFragmentPairs() {
 	}
 
 	public Optional<String> getValueOfFragmentPairKey(String key) {
-		return fragmentPairs.stream().filter(pair -> pair.fragmentKey().equals(key)).map(FragmentPair::fragmentValue)
+		return fragmentPairs.stream().filter(pair -> pair.fragmentKey().equals(key))
+				.map(FragmentPair::fragmentValue)
 				.findFirst();
 	}
 
```

Comment:
Since you changed 'getFragmentId' -> 'asString'. Perhaps you can do the same for the constructor factory method? 'fromFragmentId' -> 'fromString' ?